### ISSUE TITLE
Dev Environments — Unable to use without updating Docker 

### ISSUE THREAD

**Issue by JWhiteUX** [docker/dev-environments#62]
**Describe the bug**
tldr; Docker Desktop throws notifications without offering resolution. After running out of places to look in the desktop app, writing this ticket… I observed the monochromatic Docker Menu Icon displaying an exclamation mark. In the midst of all my app icons. Please consider the following a detailed explanation of how a product designer (👋 Me) using Docker might resolve this issue. Hope this helps future users. Please see additional remarks for more details, [Figma File (Source)](https://www.figma.com/file/9OSmJt2v7wkIydr2d5nr6l/Docker-BugTicket-UpdateDocker?node-id=0%3A1), and example solutions.

Best regards

----
Docker Dev Environments suggest users need to update to use. The banner notifications appear twice, lack resolution instructions, and/or provide unclear methods to trigger download and install of update. This results in quite a bit of chicken pecking, exploration, and subsequent dead ends in commonly recognized locations.

**To Reproduce**
1. Using Side Menu click on Dev Environments
2. Observe double-banner-notifications
3. Click on banners "please update Docker Desktop to continue…"
4. Upon failure to trigger an update, proceed to step 5
5. Close both banner notifications via `X` icon on the right
6. Restart docker… Hoping to retrigger Automatic Update
7. Observe missing resolution instructions and proceed to several more common locations in search of a solution… (See tldr; above)

**Expected behavior**
A clear and concise method to update Docker Desktop via any number of the following methods:
* Clicking on "Update docker now" on the Dev Environments Page
* Clicking on "Please update Docker Desktop…" banner notification
* In preferences > General > Under Automatically check for updates -  Click "Check for updates now"
* After launching Docker Desktop display "Update docker now" banner, click "Start Update" 
* Clicking the "bug icon" (i.e Troubleshoot) Click "Check for updates"
* Via Apple Menu > Docker Desktop > About Docker Desktop - Click "Update" - Observe version, to proceed… to google how to update docker desktop.

**Screenshots**
In no particular order, the following screenshots are the state and possible locations where I was hoping to find a method to update docker.
<img width="255" alt="Screen Shot 2021-09-22 at 2 06 18 PM" src="https://user-images.githubusercontent.com/12961108/134423254-7d7200d6-c229-4789-9e33-75aeb430fa3e.png">
<img width="1282" alt="Screen Shot 2021-09-22 at 1 46 55 PM" src="https://user-images.githubusercontent.com/12961108/134423257-5f1e8c98-a90c-40d4-b074-10991df8d929.png">
<img width="348" alt="Screen Shot 2021-09-22 at 1 45 54 PM" src="https://user-images.githubusercontent.com/12961108/134423268-5db18d9c-b2da-47f5-ac4b-58ce021fcc09.png">
<img width="1276" alt="Screen Shot 2021-09-22 at 1 45 01 PM" src="https://user-images.githubusercontent.com/12961108/134423270-9ab7aa9d-9b71-4722-b5dd-0cb209877700.png">
<img width="1280" alt="Screen Shot 2021-09-22 at 1 44 51 PM" src="https://user-images.githubusercontent.com/12961108/134423274-d7d0ee3c-25e7-40b5-9848-8b50e13861d2.png">
<img width="1276" alt="Screen Shot 2021-09-22 at 1 44 42 PM" src="https://user-images.githubusercontent.com/12961108/134423278-4b7d6ab6-43b3-450b-a4e0-517b9b7e71a2.png">

**Desktop (please complete the following information):**
 - OS: Mac BigSur (😞)
 - Version 11.2.3 (20D91)

**Version of Docker Desktop:**  
You can find the version by clicking on the `About Docker Desktop` menu
```
Version 3.5.2 (3.5.2.18)
```

**Additional context**
After writing all of the above… I noticed that the Docker Menu Icon had an exclamation mark and a quick command (CMD+R) to trigger an update.
![image](https://user-images.githubusercontent.com/12961108/134424165-bc372842-35c6-47df-8758-1121b48b3890.png)
![image](https://user-images.githubusercontent.com/12961108/134424212-ee309581-3edf-4a65-9503-ecd43d415ec1.png)

After going through all of the above, I jumped into Figma to offer some additional context and example solutions.
[Figma File (Source)](https://www.figma.com/file/9OSmJt2v7wkIydr2d5nr6l/Docker-BugTicket-UpdateDocker?node-id=0%3A1)

![image](https://user-images.githubusercontent.com/12961108/134435102-08207361-7ae7-4eda-b7c5-f2656a020eb6.png)
![image](https://user-images.githubusercontent.com/12961108/134435114-b7beb2f9-0592-4602-9e89-d488127fbb9b.png)

**Comment by felipecruz91** [comment]
Hey @JWhiteUX,

Thank you very much for your feedback. We really appreciate the time and effort you took with the designs. We'll take a look at your proposition!
